client/my-sites/stats/stats-date-picker/index.jsx

```diff
@@ -48,14 +53,10 @@ module.exports = React.createClass({
 	},
 
 	render: function() {
-		debug( 'Rendering stats date picker' );
-
-		var date = this.dateForDisplay(),
-			period = ( <span className="period"><span className="date">{ date }</span></span> ),
-			sectionTitle,
-			dateDisplay;
+		const date = this.dateForDisplay();
+		const period = ( <span className="period"><span className="date">{ date }</span></span> );
```

Comment:
since `date` is only used here, maybe just put `this.dateForDisplay()` inline here and remove the const in line 56